### PR TITLE
added new strings & replaced some strings

### PR DESCRIPTION
**Pull request by d43m0nhLInt3r**

### CODE REVIEW

File: static/locales/de-DE.yaml [hunk 1]
```diff
@@ -97,7 +97,7 @@ User Playlists:
 History:
   # On History Page
   History: Verlauf
-  Watch History: Schaue Verlauf
```
Comment (GilgusMaximus) [inline static/locales/de-DE.yaml:100]: Here it depends on whether this is for a button which starts playing all the videos in the history or whether it is a button to say "Take a look at your history". I assumed the first one.

File: static/locales/de-DE.yaml [hunk 2]
```diff
@@ -155,7 +155,7 @@ Settings:
   Player Settings:
     Player Settings: Videoplayer Einstellungen
     Force Local Backend for Legacy Formats: Erzwinge lokales System für Legacyformat
-    Remember History: Speichere Verlauf
+    Remember History: Speicher Verlauf
```
Comment (GilgusMaximus) [inline static/locales/de-DE.yaml:158]:
```suggestion
    Remember History: Verlauf speichern
```
Might be even better and more like e.g. Browsers .

File: static/locales/de-DE.yaml (hunk 1 above)
Comment (d43m0nhLInt3r) [reply static/locales/de-DE.yaml:100]: I think "Verlauf anschauen" sounds better and can be understood ambiguously

Comment (d43m0nhLInt3r) [reply static/locales/de-DE.yaml:100]: thx for your help :smile:

**Comment by PrestonN** [comment]
Appoved by @GilgusMaximus and so I will go ahead and merge.

**Pull request merged**